assets/scss/learndash-styling.scss

```diff
@@ -3,12 +3,14 @@
     color: $color_nhsuk-blue;
   }
 
-  .ld-icon.ld-primary-background {
+  .ld-icon.ld-primary-background, .ld-expand-button.ld-button-alternate .ld-icon {
```

Comment:
Being super picky that might be perfectly valid css but not a correct collapsible scss structure.
I can see `.ld-item-list-actions .ld-expand-button` and `.ld-expand-button` being used quote a lot in different context while it should be defined only once in the css hierarchy ... Any chance for a code reedit while it is still pretty hot ?